### PR TITLE
TR-11332 Add eslint-config-travix to frint

### PR DESCRIPTION
**Pull request by markvincze**
`eslint-config-travix` added, and the code issues fixed.

### CODE REVIEW

File: test/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "func-names": 0,
+    "import/no-extraneous-dependencies": 0
```
Comment (markvincze) [inline test/.eslintrc]: This is added so we can have
```
function () { ... }
```
syntax in the tests, and to be able to have `devDependencies`.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": [
+    "travix"
```
Comment (fahad19) [inline .eslintrc]: shall we import `travix/react` too?

Comment (fahad19) [inline .eslintrc]: index.js already has react. ignore my comment :D

**Comment by fahad19** [comment]
very well done @markvincze! 👏  💯

**Comment by fahad19** [comment]
can you please also add `npm run lint` to `.travis.yml` file?

https://github.com/Travix-International/frint/blob/master/.travis.yml#L8

**Pull request merged**